jwst/coron/klip.py

```diff
@@ -1,43 +1,47 @@
 """
-    Python implementation of the KLIP algorithm based on the
-    Mathematica script from Remi Soummer.
-
-:Authors: Mihai Cara
+Python implementation of the KLIP algorithm.
 
+Based on the Mathematica script from Remi Soummer.
```

Comment:
Unless we have link to that script or it is published in a paper somewhere (including ISR or TIR), this sentence is not very useful as public facing API doc.

Reply:
Removed in the latest commit